metric-anomaly-data-model/src/main/avro/AlertTask.avdl

```diff
@@ -6,6 +6,7 @@ protocol AlertTaskProtocol {
     long last_execution_time;
     string event_condition_id;
     string event_condition_type;
+    string channelId;
```

Comment:
nit: keep the new value at the end. In the next PR, let's make all fields backward compatible.